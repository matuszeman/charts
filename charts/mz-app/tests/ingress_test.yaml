suite: ingress
values:
  - ../values-ci.yaml
templates:
  - ingress.yaml
tests:
  - it: should match snapshot with values-ci.yaml
    asserts:
      - matchSnapshot: {}
