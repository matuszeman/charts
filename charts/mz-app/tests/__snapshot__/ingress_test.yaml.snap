should match snapshot with values-ci.yaml:
  1: |
    apiVersion: v1
    data:
      SCRIPT_FILENAME: /html$fastcgi_script_name
    kind: ConfigMap
    metadata:
      name: RELEASE-NAME-resource-suffix-ingress-fcgi
  2: |
    apiVersion: networking.k8s.io/v1
    kind: Ingress
    metadata:
      annotations:
        cert-manager.io/cluster-issuer: default
        nginx.ingress.kubernetes.io/auth-realm: Authentication Required
        nginx.ingress.kubernetes.io/auth-secret: test-basic-auth
        nginx.ingress.kubernetes.io/auth-type: basic
        nginx.ingress.kubernetes.io/backend-protocol: FCGI
        nginx.ingress.kubernetes.io/fastcgi-index: index.php
        nginx.ingress.kubernetes.io/fastcgi-params-configmap: RELEASE-NAME-resource-suffix-ingress-fcgi
        nginx.ingress.kubernetes.io/rewrite-target: /api/$2
        nginx.ingress.kubernetes.io/use-regex: true
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: app-name
        app.kubernetes.io/version: 0.1.0
        helm.sh/chart: mz-app-0.1.0
      name: RELEASE-NAME-resource-suffix
    spec:
      ingressClassName: nginx-external
      rules:
        - host: test.matuszeman.dev
          http:
            paths:
              - backend:
                  service:
                    name: RELEASE-NAME-resource-suffix
                    port:
                      name: app
                path: /somepath(/|$)(.*)
                pathType: ImplementationSpecific
      tls:
        - hosts:
            - test.matuszeman.dev
          secretName: RELEASE-NAME-resource-suffix-ingress-tls
